Server.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const app = express();
+const port = 3000;
+
+// define the ping route
+app.get("/ping",(req,res)=>{
```

Comment:
Consider adding a default route to handle undefined routes. This can improve user experience by providing a meaningful response when an undefined route is accessed. [medium]